pkg/provider/bitbucketcloud/acl.go

```diff
@@ -70,14 +71,10 @@ func (v *Provider) checkMember(event *info.Event) (bool, error) {
 }
 
 func (v *Provider) checkOkToTestCommentFromApprovedMember(event *info.Event) (bool, error) {
-	prNumber, err := v.getPullRequestNumber(event.Event)
```

Comment:
I am not sure I understand this change, is this related? is PullRequestNumber a unsigned integer  now?

Reply:
instead of getting from event interface ... just getting the pull request number from `Event` obj where it is already there
while parsing the payload